std/meta.d

```diff
@@ -61,6 +61,67 @@ unittest
 }
 
 /**
+ * Not everything can be directly aliased. An alias cannot be declared
+ * of - for example - a literal:
+ * values:
+ *
+ * `alias a = 4; //Error`
```

Comment:
I'm currently getting back into some Phobos development and PR reviewing after a long absence. Do we have Markdown-style inline code formatting now?